app/packages/looker/src/lookers/imavid/index.ts

```
@@ -200,4 +229,70 @@ export class ImaVidLooker extends AbstractLooker<ImaVidState, Sample> {
       this.updater({ options, disabled: false });
     }
   }
+
+  refreshOverlaysToCurrentFrame() {
+    let { image: _cachedImage, ...thisFrameSample } =
+      this.frameStoreController.store.getSampleAtFrame(
+        this.frameNumber
+      )?.sample;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)